Svg.Editor.Core/SvgDrawingCanvas.cs

```diff
@@ -312,18 +312,24 @@ public async Task OnGesture(UserGesture gesture)
 			}
 		}
 
-		/// <summary>
-		/// Called by platform specific implementation to allow tools to draw something onto the canvas
-		/// </summary>
-		/// <param name="renderer"></param>
-		public async Task OnDraw(IRenderer renderer)
+        public Action<IRenderer> OnDrawAction;
```

Comment:
You are missing the `event` keyword here.
Also, though Actions are cool, please define an old fashioned Delegate.
https://learn.microsoft.com/en-us/dotnet/standard/events/

The advantage of delegates is, that you can _name_ the arguments.
For an action this is always item1, item2,...
So a delegate is much better for an event as it can be seen as an interface